rgw/v2/tests/s3_swift/configs/test_s3select_query_gen_csv_depth1.yaml

```diff
@@ -0,0 +1,10 @@
+# script: test_s3select.py
```

Comment:
Please update the polarion ID as well in both configs

Reply:
@anrao19 need to create/modify polarion testcases according to the code we develop
will update all tests with polarion ids in the coming PR's